screens/Play/index.tsx

```diff
@@ -29,17 +31,28 @@ interface Props {
 export default function PlayScreen({ route, navigation }: Props) {
   const { id } = route.params
   const meditation = useMeditation(id)
+  const favourites = useAppSelector(selectFavourites)
   const [isLoadingAudio, setIsLoadingAudio] = React.useState(true)
   const [isPlaying, setIsPlaying] = React.useState(false)
   const [sound, setSound] = React.useState<Audio.Sound>()
   const [positionMillis, setPositionMillis] = React.useState(0)
   const [durationMills, setDurationMills] = React.useState(0)
+  const [isfavourited, setIsFavourited] = React.useState(false)
   const durationTime = useMsToTime(durationMills)
   const positionTime = useMsToTime(positionMillis)
   const dispatch = useAppDispatch()
   const uri = meditation?.uri || ''
   const filepaths = useAppSelector(selectFilePaths)
 
+  React.useEffect(() => {
+    setIsFavourited(favourites.findIndex((item: Meditation) => item.id === meditation?.id) !== -1)
+  }, [favourites, meditation?.id])
```

Comment:
You can probably just rely on redux to update the component instead of copying things to local state like this:

```suggestion
const isFavourited = favourites.findIndex((item: Meditation) => item.id === meditation?.id) !== -1
```